src/components/auth/auth.service.ts

```diff
@@ -184,6 +188,93 @@ export class AuthService {
     return session;
   }
 
+  async forget(input: UserEmailInput): Promise<boolean> {
+    const token = this.encodeJWT();
+    const email = input.email;
+
+    await this.db
+      .query()
+      .raw(
+        `
+      CREATE(et:EmailToken{value:$value, token: $token, createdOn:datetime()})
+      RETURN et as emailToken
+      `,
+        {
+          $value: email,
+          $token: token,
+        }
+      )
+      .first();
+    const params = {
+      Destination: { ToAddresses: [email] },
+      Message: {
+        Body: {
+          Html: {
+            Charset: 'UTF-8',
+            Data: `<html><body><p>This is your secret login code:</p>
+                          <a href="http://localhost:3333/auth/reset?token=${token}">Go to Login</a></body></html>`,
```

Comment:
We'll want to have the frontend url be in the config service so it can different deployment environments reference their own UI. Something like `resetPasswordUrl`